#!/usr/bin/env python
# coding: utf-8

# In[1]:


>> icecream = {'폴라포': 1200, '빵빠레': 1800, '월드콘': 1500, '메로나': 1000}
>> icecream['누가바']
Traceback (most recent call last):
  File "<pyshell#69>", line 1, in <module>
    icecream['누가바']
    
# 딕셔너리 내에 있는 키값을 호출하지 않았기 때문에 오류가 발생

